recipe/conda_build_config.yaml

```diff
@@ -6,5 +6,3 @@ docker_image:
   - nsls2/nsls2forge:latest
 cuda_compiler_version:
   - None
-python:
-  - 3.6
```

Comment:
Please copy https://github.com/nsls-ii-forge/event-model-feedstock/blob/master/recipe/conda_build_config.yaml and rerender. This will enable the auto-tick bot to process this repo correctly.

Reply:
Done.